contracts/drops/token/HolographDropsEditionsV1.sol

```diff
@@ -429,8 +436,9 @@ contract HolographDropsEditionsV1 is NonReentrant, ERC721H, IHolographERC721Drop
       revert Presale_MerkleNotApproved();
     }
 
-    if (msg.value != pricePerToken * quantity) {
-      revert Purchase_WrongPrice(pricePerToken * quantity);
+    uint256 weiPrice = _usdToWei(pricePerToken);
+    if (msg.value < weiPrice * quantity) {
```

Comment:
maybe just to be specific use `weiPricePerToken`?

Reply:
Sure